### ISSUE TITLE
CodeBuild second example template fails

### ISSUE THREAD

**Issue by pofallon** [awsdocs/aws-cloudformation-user-guide#54]
The second CodeBuild sample JSON template defines a source of `CODEPIPELINE` and `"BadgeEnabled": "true"`.  However, trying to launch a stack with this combination of properties fails with the error:
```
Build badges are not supported for CodePipeline source (Service: AWSCodeBuild; Status Code: 400; Error Code: InvalidInputException; Request ID: ... )
```

**Comment by wberrycdoor** [comment]
+1

**Comment by derdeka** [comment]
+1

**Comment by ismarslomic** [comment]
+1

**Comment by ckesser** [comment]
Many apologies for the very, VERY late response to this.

We removed the BadgeEnabled specification from the second example here: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-codebuild-project.html. We also added a note to the BadgeEnabled property description clarifying that build badges aren't supported for source CODEPIPELINE. 

I'm going to close out this issue as resolved. Thank you so much for bringing this to our attention!
